test/integration/ttlcontroller/ttlcontroller_test.go

```diff
@@ -60,27 +61,48 @@ func createNodes(t *testing.T, client *clientset.Clientset, startIndex, endIndex
 					Name: fmt.Sprintf("node-%d", idx),
 				},
 			}
-			if _, err := client.CoreV1().Nodes().Create(context.TODO(), node, metav1.CreateOptions{}); err != nil {
-				t.Fatalf("Failed to create node: %v", err)
+			_, err := client.CoreV1().Nodes().Create(context.TODO(), node, metav1.CreateOptions{})
+			if err != nil {
+				errs <- err
 			}
 		}(i)
 	}
-	wg.Wait()
+
+	go func() { // wait in another go-routine to close channel
+		wg.Wait()
+		close(errs)
+	}()
+
+	for err := range errs {
```

Comment:
Don't we need to add `if err != nil {` like line 102?

Reply:
Yes, thanks for pointing out. Let me make that change.

Reply:
Have changed that in the latest commit.